Xamarin/DrawXShared/DrawXSettingsManager.cs

```diff
@@ -60,20 +41,43 @@ public static DrawXSettings Settings
                 {
                     Write(() =>
                     {
-                        _savedSettings = new DrawXSettings { LastColorUsed = "Indigo" };
-                        _localSettingsRealm.Add(_savedSettings);
+                        _savedSettings = _localSettingsRealm.Add(new DrawXSettings
+                        {
+                            LastColorUsed = "Indigo"
+                        });
                     });
                 }
 
                 return _savedSettings;
             }
         }
 
+        public static User CurrentUser
+        {
+            get
+            {
+                try
+                {
+                    return User.Current;
+                }
+                catch
+                {
+                    // We have ended up with more than one logged in user. Log all of them out.
```

Comment:
I used to be able to repro it on Android by quickly clicking the logout button. With the latest change on [`EditCredentials`](https://github.com/realm-demos/realm-draw/pull/14/files#diff-59c37b0b71867ba147b393a8c86d11c0R152), it has gone away, but I forgot to remove this code.